lib/state/appActions.js

```diff
@@ -85,15 +92,19 @@ appActions.load = function(cb) {
 
     async.series([
       api.user.account.bind(null),
-      api.user.profile.bind(null)
+      api.user.profile.bind(null),
+      api.user.getUploadGroups.bind(null)
     ], function(err, results) {
       var account = results[0];
       var profile = results[1];
-      var user = _.assign({}, account, {profile: profile});
+      var uploadGroups = results[2];
+      console.log(uploadGroups);
+      var user = _.assign({}, account, {profile: profile, uploadGroups: uploadGroups});
 
+      // todo add state for when the user cannot upload to any account
       self.app.setState({
         user: user,
-        targetId: user.userid,
+        targetId: uploadGroups[0].userid,
```

Comment:
seems odd? so the first user in uploadGroups is always there and its the logged in user?